compiler/statements.go

```diff
@@ -25,7 +29,8 @@ const (
 )
 
 type Statement struct {
-	Type StatementType
+	rowToInsert Row
+	Type        StatementType
```

Comment:
`RowToInsert`* , struct fields which don't start with capital letters are not exportable. Let's stick with this format for struct fields as well